src/main/resources/anet-schema.yml

```diff
@@ -461,56 +461,38 @@ properties:
 
       person:
         type: object
-        additionalProperties: false
```

Comment:
Why did you remove this? It was there so admins don't add non-supported properties wihout being warned about this.

Reply:
I did remove because custom fields were lifted, so there were additional properties

Reply:
Another reason not to do it like this, but to do something like https://github.com/NCI-Agency/anet/pull/3278#discussion_r517923184